src/Collection/Collection.service.ts

```diff
@@ -244,21 +221,16 @@ export class CollectionService {
     return false
   }
 
-  public async getDbTPWCollections(
-    manager: string
+  // TODO: rename by getDbTPCollectionsByThirdParties?
+  public async getDbTPCollections(
```

Comment:
Why did this method change to receive third parties? It requires the user of this method to request the third parties in advance, making it more complex to use IMO.

Reply:
We talked a bit about this at the time, but it was necessary when we were returning managers. No longer necessary (for now at least). I'll change it!